diff -Nur Phantom-Satellite-33.7.1/platform/security/nss/lib/freebl/Makefile Phantom-Satellite-Dev/platform/security/nss/lib/freebl/Makefile
--- Phantom-Satellite-33.7.1/platform/security/nss/lib/freebl/Makefile	2025-05-09 14:45:49.000000000 -0500
+++ Phantom-Satellite-Dev/platform/security/nss/lib/freebl/Makefile	2025-05-18 22:50:18.717256769 -0500
@@ -785,11 +785,11 @@
 $(OBJDIR)/$(PROG_PREFIX)chacha20poly1305-ppc$(OBJ_SUFFIX): CFLAGS += -maltivec
 endif
 ifneq ($(NSS_DISABLE_CRYPTO_VSX),1)
-$(OBJDIR)/$(PROG_PREFIX)gcm-ppc$(OBJ_SUFFIX): CFLAGS += -mcrypto -mvsx
-$(OBJDIR)/$(PROG_PREFIX)gcm$(OBJ_SUFFIX): CFLAGS += -mcrypto -mvsx
-$(OBJDIR)/$(PROG_PREFIX)rijndael$(OBJ_SUFFIX): CFLAGS += -mcrypto -mvsx
-$(OBJDIR)/$(PROG_PREFIX)sha512$(OBJ_SUFFIX): CFLAGS += -mcrypto -mvsx
-$(OBJDIR)/$(PROG_PREFIX)chacha20poly1305-ppc$(OBJ_SUFFIX): CFLAGS += -mcrypto -mvsx
+$(OBJDIR)/$(PROG_PREFIX)gcm-ppc$(OBJ_SUFFIX): CFLAGS += -mno-crypto -mvsx
+$(OBJDIR)/$(PROG_PREFIX)gcm$(OBJ_SUFFIX): CFLAGS += -mno-crypto -mvsx
+$(OBJDIR)/$(PROG_PREFIX)rijndael$(OBJ_SUFFIX): CFLAGS += -mno-crypto -mvsx
+$(OBJDIR)/$(PROG_PREFIX)sha512$(OBJ_SUFFIX): CFLAGS += -mno-crypto -mvsx
+$(OBJDIR)/$(PROG_PREFIX)chacha20poly1305-ppc$(OBJ_SUFFIX): CFLAGS += -mno-crypto -mvsx
 endif
 endif
 
diff -Nur Phantom-Satellite-33.7.1/platform/security/nss/lib/freebl/freebl.gyp Phantom-Satellite-Dev/platform/security/nss/lib/freebl/freebl.gyp
--- Phantom-Satellite-33.7.1/platform/security/nss/lib/freebl/freebl.gyp	2025-05-09 14:45:49.000000000 -0500
+++ Phantom-Satellite-Dev/platform/security/nss/lib/freebl/freebl.gyp	2025-05-18 22:50:18.809257830 -0500
@@ -299,11 +299,11 @@
       'conditions': [
         [ 'disable_crypto_vsx==0', {
           'cflags': [
-            '-mcrypto',
+            '-mno-crypto',
             '-maltivec'
            ],
            'cflags_mozilla': [
-             '-mcrypto',
+             '-mno-crypto',
              '-maltivec'
            ],
         }, 'disable_crypto_vsx==1', {
@@ -376,14 +376,14 @@
       'conditions': [
         [ 'disable_crypto_vsx==0', {
           'cflags': [
-            '-mcrypto',
+            '-mno-crypto',
             '-maltivec',
             '-mvsx',
             '-funroll-loops',
             '-fpeel-loops'
            ],
            'cflags_mozilla': [
-            '-mcrypto',
+            '-mno-crypto',
             '-maltivec',
             '-mvsx',
             '-funroll-loops',
@@ -415,14 +415,14 @@
       'conditions': [
         [ 'disable_crypto_vsx==0', {
           'cflags': [
-            '-mcrypto',
+            '-mno-crypto',
             '-maltivec',
             '-mvsx',
             '-funroll-loops',
             '-fpeel-loops'
            ],
            'cflags_mozilla': [
-            '-mcrypto',
+            '-mno-crypto',
             '-maltivec',
             '-mvsx',
             '-funroll-loops',
