diff --git a/media/ffvpx/libavcodec/flacdec.c b/media/ffvpx/libavcodec/flacdec.c
index ed2de14d0a..efee47e35b 100644
--- a/media/ffvpx/libavcodec/flacdec.c
+++ b/media/ffvpx/libavcodec/flacdec.c
@@ -144,34 +144,38 @@ static int allocate_buffers(FLACContext *s)
 
     buf_size = av_samples_get_buffer_size(NULL, s->stream_info.channels,
                                           s->stream_info.max_blocksize,
                                           AV_SAMPLE_FMT_S32P, 0);
     if (buf_size < 0)
         return buf_size;
 
     av_fast_malloc(&s->decoded_buffer, &s->decoded_buffer_size, buf_size);
-    if (!s->decoded_buffer)
+    if (!s->decoded_buffer) {
+        memset(s->decoded, 0, sizeof(s->decoded));
         return AVERROR(ENOMEM);
+    }
 
     ret = av_samples_fill_arrays((uint8_t **)s->decoded, NULL,
                                  s->decoded_buffer,
                                  s->stream_info.channels,
                                  s->stream_info.max_blocksize,
                                  AV_SAMPLE_FMT_S32P, 0);
     if (ret >= 0 && s->stream_info.bps == 32 && s->stream_info.channels == 2) {
         buf_size = av_samples_get_buffer_size(NULL, 1,
                                               s->stream_info.max_blocksize,
                                               AV_SAMPLE_FMT_S64P, 0);
         if (buf_size < 0)
             return buf_size;
 
         av_fast_malloc(&s->decoded_buffer_33bps, &s->decoded_buffer_size_33bps, buf_size);
-        if (!s->decoded_buffer_33bps)
+        if (!s->decoded_buffer_33bps) {
+            s->decoded_33bps = NULL;
             return AVERROR(ENOMEM);
+        }
 
         ret = av_samples_fill_arrays((uint8_t **)&s->decoded_33bps, NULL,
                                      s->decoded_buffer_33bps,
                                      1,
                                      s->stream_info.max_blocksize,
                                      AV_SAMPLE_FMT_S64P, 0);
 
     }
@@ -670,17 +674,17 @@ static int decode_frame(FLACContext *s)
         av_log(s->avctx, AV_LOG_ERROR, "sample rate not found in STREAMINFO"
                                         " or frame header\n");
         return AVERROR_INVALIDDATA;
     }
     if (fi.samplerate == 0)
         fi.samplerate = s->stream_info.samplerate;
     s->stream_info.samplerate = s->avctx->sample_rate = fi.samplerate;
 
-    if (!s->got_streaminfo) {
+    if (!s->got_streaminfo || !s->decoded_buffer) {
         ret = allocate_buffers(s);
         if (ret < 0)
             return ret;
         s->got_streaminfo = 1;
         dump_headers(s->avctx, &s->stream_info);
     }
     ff_flacdsp_init(&s->dsp, s->avctx->sample_fmt,
                     s->stream_info.channels);
